envoy/upstream/load_balancer.h

```diff
@@ -83,6 +83,16 @@ class LoadBalancerContext {
    * Returns the transport socket options which should be applied on upstream connections
    */
   virtual Network::TransportSocketOptionsConstSharedPtr upstreamTransportSocketOptions() const PURE;
+
+  using OverrideHostStatus = uint32_t;
```

Comment:
This still needs documentation on how the bitfield works. It's not clear from reading this.